cmake/Corrosion.cmake

```diff
@@ -1690,6 +1692,9 @@ function(corrosion_add_cxxbridge cxx_target)
             $<INSTALL_INTERFACE:include>
     )
 
+    if(DEFINED _arg_REGEN_TARGET)
```

Comment:
I guess this empty if was accidentally committed?

Reply:
Ah, yes, will fix.